HelpSource/Classes/Stethoscope.schelp

```diff
@@ -172,6 +172,23 @@ METHOD:: style
         One of the above Integers.
 
 
+SUBSECTION:: Size and position
+        
+METHOD:: bounds
+    The position and size of the window. The position is relative to the bottom-left corner of the screen.
+    argument::
+        A Rect, a Point interpreted link::Classes/Point#-asRect#.asRect:: or anything that responds to it.
```

Comment:
Using`Point` perhaps isn't the best example since, although that's valid, specifying a Point implicitly sets the position to `[0,0]`. So a different example would illustrate the the position _and_ size are set by `bounds_`.

Something like:
>A link::Classes/Rect::, or any object responding to the link::Search#-asRect:: method, such as an link::Classes/Array:: or A link::Classes/Size::.

Reply:
have done!
https://github.com/supercollider/supercollider/pull/6281/commits/39802c61144acfa132c8c72b41da4d118490cddb